include/cuco/detail/static_multimap/static_multimap.inl

```diff
@@ -336,7 +326,6 @@ OutputIt static_multimap<Key, Value, Scope, Allocator, ProbeSequence>::retrieve_
 
   // Using per-warp buffer for vector loads and per-CG buffer for scalar loads
   constexpr auto buffer_size = uses_vector_load() ? (warp_size() * 3u) : (cg_size() * 3u);
```

Comment:
```suggestion
  constexpr auto buffer_size = this->uses_vector_load() ? (this->warp_size() * 3u) : (this->cg_size() * 3u);
```

I'm seeing this now but it's no subject for this PR. We can defer this to a separate cleanup PR.

Reply:
Also, `warp_size()` feels like something that should be defined in `cuco/detail/utility/cuda.hpp`